luxjs/forms/form.js

```diff
@@ -309,6 +314,10 @@
                     // Add model attribute
                     input.attr('ng-model', scope.formModelName + '["' + field.name + '"]');
 
+                    // Add default placeholder to date field
+                    if (field.type === 'date')
+                        field.placeholder = 'YYYY-MM-DD';
```

Comment:
You are overriding the placeholder here, what about if it was already available?